upstream/controllers.go

```diff
@@ -136,6 +145,8 @@ func PullCanaries(c echo.Context) error {
 		if err != nil {
 			return c.JSON(http.StatusBadRequest, api.HTTPError{Error: err.Error(), Message: "'since' param needs to be a valid RFC3339 timestamp"})
 		}
+
+		ctx.SetSpanAttributes(attribute.String("since", sinceRaw))
```

Comment:
prefix this with `upstream.since` since these attributes will be at global scope